public/app/common/components/cc-form/cc-form.jsx

```diff
@@ -0,0 +1,36 @@
+var React = require('react');
+var styleMixin = require('mixins/style-mixin');
+var t = require('tcomb-form');
```

Comment:
It's library that I used for forms, it seems to be the best fit for our project. http://gcanti.github.io/tcomb-form/guide/index.html#hello-form